flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -362,7 +361,9 @@ public void start() throws Exception {
 					resourceManagerGatewayRetriever,
 					blobServer.getTransientBlobService(),
 					executor,
-					MetricFetcherImpl.fromConfiguration(
+					updateInterval == 0 ?
```

Comment:
I would prefer introducing a dedicated variable for the `MetricFetcher`, i.e.,

```java
MetricFetcher metricFetcher = updateInterval == 0
                        ? VoidMetricFetcher.INSTANCE
					   : MetricFetcherImpl.fromConfiguration(...)
```

this should improve code readability.

Reply:
@TisonKun  thanks for your reviewing. Updated this pr to address your comment.